src/components/StaffComments/StaffComments.vue

```diff
@@ -0,0 +1,252 @@
+<template>
```

Comment:
This is exact same component that you already reviewed in Staff Comments Part 1 PR.

Except... I had to remove conditional chaining (`a?.b`) as that caused a JS error. This is likely due to the TS version used but I can find nothing out of date.

This can be fixed in the future.